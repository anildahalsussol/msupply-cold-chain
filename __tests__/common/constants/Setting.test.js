import { SETTING } from '~constants';

describe('Constants: SETTING', () => {
  it('Matches snapshot', () => {
    expect(SETTING).toMatchSnapshot();
  });
});
