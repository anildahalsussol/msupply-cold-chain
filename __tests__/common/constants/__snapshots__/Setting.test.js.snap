// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: SETTING Matches snapshot 1`] = `
Object {
  "BLUETOOTH": Object {
    "DEFAULT_LOG_INTERVAL": "DEFAULT_LOG_INTERVAL",
  },
  "SYNC": Object {
    "AUTH_PASSWORD": "authPassword",
    "AUTH_URL": "authUrl",
    "AUTH_USERNAME": "authUsername",
    "IS_INTEGRATING": "isIntegrating",
    "IS_PASSIVE_SYNC_ENABLED": "isPassiveSyncEnabled",
    "LAST_SYNC": "lastSync",
    "SENSOR_URL": "sensorUrl",
    "TEMPERATURE_BREACH_URL": "temperatureBreachUrl",
    "TEMPERATURE_LOG_URL": "temperatureLogUrl",
  },
}
`;
